pulsar-client-kafka-compat/pulsar-client-kafka/src/main/java/org/apache/kafka/clients/consumer/PulsarKafkaConsumer.java

```diff
@@ -388,12 +399,12 @@ public void seek(TopicPartition partition, long offset) {
 
     @Override
     public void seekToBeginning(Collection<TopicPartition> partitions) {
-        throw new UnsupportedOperationException();
+//        throw new UnsupportedOperationException();
```

Comment:
Why comment it if the implementation is not there?

Reply:
Ouch, this is unrelated to this change. Will remove.